python_modules/jupedsim/tests/test_distributions.py

```diff
@@ -1,5 +1,7 @@
-from ..jupedsim import distributions
+from jupedsim import distributions
 import pytest
+# tests can be called from \python_modules\jupedsim
+# command to call test: python -m pytest -vv .\tests\test_distributions.py
```

Comment:
This is better suited for the project readme or an alternative readme specific to the python_modules.

Additionally it should read:
from `jpscore/python_modules/jupedsim` you can call `python -m pytest`

For an explanation please see:

See: 
https://doc.pytest.org/en/latest/explanation/goodpractices.html#choosing-a-test-layout-import-rules

and 

https://doc.pytest.org/en/latest/explanation/pythonpath.html#pytest-vs-python-m-pytest

Reply:
readme was created inside jpscore/python_modules/jupedsim.